### PR TITLE
Convert to Dev Mode

### PR DESCRIPTION
**Pull request by jj-zhang**
issue: https://github.com/OpenLiberty/guide-jpa-intro/issues/135

### CODE REVIEW

**Comment by gkwan-ibm** [comment]
- May need more investigation for copying the derby library to the shared directory

**Comment by gkwan-ibm** [comment]
@jj-zhang I updated the pom.xml files and the travis tests. Now the app can run as our expectation for dev mode. You may resume this work to update the README:
- should run and stop the app by 2 command line sessions (see how the opentracing guide does in the lgdev)
- start dev mode by 2 command line sessions 
- at "Running the application" section, move the stop instruction after  the running the tests (i.e. at the end of the guide)
- at "Running the tests", make clear to the readers which shell session to hit enter to run the test because there are 2 sessions
- find out any other update

**Comment by gkwan-ibm** [comment]
@jj-zhang 

- pom.xml
  - [ ] update liberty-maven-plugin to 3.2.2
- **Try what you’ll build**
  - [ ] the instruction to restart not clear, better to tell restart which or both. I think restart the backendServices should be enough.
  - [ ] "shell sessions" -> "command-line sessions"
  - [ ] "shell session" -> "command-line session"
- **Defining a JPA entity class**
  - [ ] "two command-line instances" -> "two command-line sessions"
  - [ ] "Open another command line to continue..." -> "Open another command-line session to continue..."
- **Running the application**
  - [ ] no need to stop servers because using dev mode
- **Testing the application**
  - EventEntityIT.java
    - [ ] format EventEntityIT.java so that no wrapped lines
    - [ ] try remove `@TestMethodOrder` because didn't use test order in the code
  - **Running the tests**
    - [ ] rewrite the instruction because frontendUI has no test, clearly to tell press enter in which command-line session
    - [ ] "shell sessions" -> "command-line sessions"
    - [ ] "modules" -> "projects"

**Comment by gkwan-ibm** [comment]
- **Try what you’ll build**
  - [ ] at the last line, "modules" -> "projects"
- **Testing the application**
  - EventEntityIT.java
    - [ ] format EventEntityIT.java so that all assertEquals do not wrap line

File: finish/backendServices/src/test/java/it/io/openliberty/guides/event/EventIT.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 // end::copyright[]
```
Comment (MaiHameed) [inline finish/backendServices/src/test/java/it/io/openliberty/guides/event/EventIT.java:12]: Update copyright year

File: start/backendServices/src/test/java/it/io/openliberty/guides/event/EventIT.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
 // end::copyright[]
```
Comment (MaiHameed) [inline start/backendServices/src/test/java/it/io/openliberty/guides/event/EventIT.java:12]: Update copyright year

File: README.adoc [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018, 2019 IBM Corporation and others.
+// Copyright (c) 2018, 2019, 2020 IBM Corporation and others.
```
Comment (MaiHameed) [inline README.adoc:1]: Should be `2018, 2020`

**Comment by gkwan-ibm** [comment]
hi @Charlotte-Holt would you review the README.adoc?

File: README.adoc [hunk 4]
```diff
@@ -80,28 +99,60 @@ event entity to the database. The event is now stored in the database and is vis
 current events.
 
 Notice that if you stop the Open Liberty server and then restart it, the events created
-are still displayed in the list of current events. Run the following Maven goals to stop
+are still displayed in the list of current events. Navigate to the `finish/backendServices` and run the following Maven goals to stop
```
Comment (Charlotte-Holt) [inline README.adoc:102]: Suggested update

>Navigate to the `finish/backendServices` and run the following Maven goals to stop and then restart the server:

to

>Navigate to the `finish/backendServices` directory and run the following Maven goals to stop
and then restart the server:

File: README.adoc [hunk 5]
```diff
@@ -80,28 +99,60 @@ event entity to the database. The event is now stored in the database and is vis
...
 The events created are still displayed in the list of current events. The `Update` action link
 located beside each event allows you to make modifications to the persisted entity and the 
 `Delete` action link allows you to remove entities from the database.
 
-[role=command]
-include::{common-includes}/trywhatyoubuild-end.adoc[]
+After you are finished checking out the application, stop the Open Liberty servers by pressing CTRL+C in the
+command-line sessions where you ran the `backendServices` and `frontendUI` services.
+Alternatively, you can run the `liberty:stop` goal from the finish directory in another command-line session for the `frontendUI`
+and `backendServices` projects:
```
Comment (Charlotte-Holt) [inline README.adoc:117]: Suggested update

>Alternatively, you can run the `liberty:stop` goal from the finish directory in another command-line session for the `frontendUI` and `backendServices` projects:

to

>Alternatively, you can run the `liberty:stop` goal from the `finish` directory in another command-line session for the `frontendUI` and `backendServices` services:

File: README.adoc [hunk 6]
```diff
@@ -80,28 +99,60 @@ event entity to the database. The event is now stored in the database and is vis
...
+----
 
 // =================================================================================================
 // Defining a JPA entity class
 // =================================================================================================
 
 == Defining a JPA entity class
 
-Navigate to the `start` directory to begin.
+You will need two command-line sessions to start the two services in development mode.
```
Comment (Charlotte-Holt) [inline README.adoc:129]: Suggested update

>You will need two command-line sessions to start the two services in development mode.

to

>You need two command-line sessions to start the two services in development mode.

File: README.adoc [hunk 7]
```diff
@@ -307,22 +351,33 @@ event is deleted and one final check is done to ensure that the updated test eve
...
 
 Results :
 
 Tests run: 5, Failures: 0, Errors: 0, Skipped: 0 
 ----
 
+When you are done checking out the services, exit development mode by pressing CTRL+C in the command-line sessions where you
+ran the `frontendUI` and `backendServices` services,  or by typing `q` and then pressing the `enter/return key`.
+Alternatively, you can run the `liberty:stop` goal from the finish directory in another command-line session for the `frontendUI`
+and `backendServices` projects:
```
Comment (Charlotte-Holt) [inline README.adoc:375]: Suggested update

>When you are done checking out the services, exit development mode by pressing CTRL+C in the command-line sessions where you ran the `frontendUI` and `backendServices` services,  or by typing `q` and then pressing the `enter/return key`. Alternatively, you can run the `liberty:stop` goal from the finish directory in another command-line session for the `frontendUI` and `backendServices` projects:

to

>When you are done checking out the services, exit dev mode. You can exit dev mode by pressing CTRL+C in the command-line sessions where you ran the `frontendUI` and `backendServices` services, or by typing `q` and then pressing the `enter/return` key. Alternatively, you can run the `liberty:stop` goal from the `finish` directory in another command-line session for the `frontendUI` and `backendServices` services:

**Comment by jj-zhang** [comment]
@Charlotte-Holt Thanks for your review! I've addressed your feedback and feel free to check it.

**Comment by DannyS03** [comment]
tested/verified on **qa**.

**Pull request merged**